### PR TITLE
Add design documents

### PR DESCRIPTION
**Pull request by qiaojialin**

### CODE REVIEW

**Comment by jixuan1989** [comment]
LGTM, but don't you need to modify the website code?

File: docs/Documentation-CHN/Developers/Architecture.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+# 应用概览
+
+![avatar](Overview.png)
+
+物联网时序数据库 Apache IoTDB 的架构图如上所示，覆盖时序数据的采集、存储、查询、分析、可视化等全生命周期，其中灰色部分为 IoTDB 组件。
```
Comment (sunjincheng121) [inline docs/Documentation-CHN/Developers/Architecture.md]: "覆盖时序数据的采集、存储、查询、分析、可视化等全生命周期，" -> "覆盖了对时序数据的采集、存储、查询、分析以及可视化等全生命周期的数据处理(或者 “管理”)功能。"?

File: docs/Documentation-CHN/Developers/Architecture.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## IoTDB 架构介绍
+
+![avatar](Architecture.png)
+
+IoTDB 采用客户端-服务器架构。其中服务器端主要包括查询引擎，用来处理用户的所有请求，并分发到对应的管理组件，包括数据写入层、数据查询、元数据管理、权限管理等模块。
```
Comment (sunjincheng121) [inline docs/Documentation-CHN/Developers/Architecture.md]: 将“IoTDB 采用客户端-服务器架构。" 移动到 架构图上面 是不是 好一点，因为目前看 标题下面直接图片，有点空。建议放到图片上方，比如： “IoTDB 采用客户端-服务器架构，如下图所示：”。

File: docs/Documentation-CHN/Developers/Architecture.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* [元数据管理](./SchemaManager/MManager.md)
+* [存储引擎](StorageEngine/StorageEngine.md)
+* [数据查询](DataQuery/DataQuery.md)
+* [权限管理](./Administration/Administration.md)
+
+## 连接器
+
+IoTDB 与大数据系统进行了对接。
+
+* [Hadoop-TsFile](./Connector/Hadoop-TsFile.md)
```
Comment (sunjincheng121) [inline docs/Documentation-CHN/Developers/Architecture.md]: 是不是也要加一下： `Grafana`?

File: docs/Documentation-CHN/Developers/DataQuery/DataQuery.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+	* 聚合查询
+	* 降采样查询
+	* 单点补空值查询
+	* 最新数据查询等
+	
+## 查询计划
+
+主要包括两种查询框架：
+
+* 不带值过滤条件的（如，select s1,s2,s3 from root.d1）
```
Comment (sunjincheng121) [inline docs/Documentation-CHN/Developers/DataQuery/DataQuery.md]: 建议 在查询的示例 里面  用 `f' 表示 Field, 就是 `s1,s2,s3` -> `f1, f2, f3`, 同时字段之间添加空格。

Comment (qiaojialin) [reply docs/Documentation-CHN/Developers/DataQuery/DataQuery.md]: 多谢，已改

File: docs/Documentation-CHN/Developers/Architecture.md (hunk 3 above)
Comment (qiaojialin) [reply docs/Documentation-CHN/Developers/Architecture.md]: 嗯对，增加了

File: docs/Documentation-CHN/Developers/Architecture.md (hunk 2 above)
Comment (qiaojialin) [reply docs/Documentation-CHN/Developers/Architecture.md]: 好的

File: docs/Documentation-CHN/Developers/Architecture.md (hunk 1 above)
Comment (qiaojialin) [reply docs/Documentation-CHN/Developers/Architecture.md]: 好的，已更新

**Comment by sunjincheng121** [comment]
Thanks for the update @qiaojialin !
Could you please squash the commits, and I would like to have an other review :)

**Comment by sunjincheng121** [comment]
I'm sorry I just came back from today's meeting.

**Comment by sunjincheng121** [comment]
> Thanks for the update @qiaojialin !
> Could you please squash the commits, and I would like to have an other review :)

I'm sorry I just came back from today's meeting. @qiaojialin :(

**Pull request merged**